.gitattributes

```diff
@@ -11,3 +11,15 @@ hw/top_*/rdc/*.tcl           cdc
 hw/rdc/tools/*/run-rdc.tcl   cdc
 hw/rdc/tools/dvsim/rdc.mk    cdc dvsim
 hw/rdc/tools/dvsim/*.hjson   cdc dvsim
+
+# doc
```

Comment:
Should we also include the root `doc` folder, and a general `*.md` rule?

Reply:
SG. Let me add that and merge